### ISSUE TITLE
installs appear in manifest.toml by install comments with improved spacing

### ISSUE THREAD

**Issue by ghudgins** [flox/flox#448]
# Acceptance criteria

* `flox install go`
* `flox edit`
result: `go` and `[install]` is at the bottom of the manifest, separate from the comment suggestion for install

desired: I see that there is a single install command with commented out suggestions and 'go' underneath the comments

<img width="496" alt="image" src="https://github.com/flox/flox/assets/5465514/edcd5f28-3b24-452e-80b5-bba57e0101e8">

**Comment by ghudgins** [comment]
from @rossturk - include an intro section and better section headings with spacing. 

```
#
# This is a flox environment manifest.
# Visit flox.dev/docs/edit or see flox-edit(1) for more information
#

# List packages you wish to install in your environment inside
# the [install] section

[install]
# hello = {}
# nodejs = { version = "^16.4.2" }
overpass = {}

# An activation hook will be run when entering the environment.
# You can define one in [hook] with an inline `script` field,
# or provide a file using the `file` field.

[hook]
# file = ~/myproject/start_env.sh
script = """
  echo "Welcome to your flox environment!";
"""

**Comment by ysndr** [comment]
The intro looks good though

**Comment by rossturk** [comment]
> I think with the latest changes, the installed things are written _above_ the comment:
> 
> ```
> [install]
> overpass = {}
> # hello = {}
> # nodejs = { version = "^16.4.2" }
> ```
> 
> writing it below the comments would be quite a bigger technical change

I think that's okay. Probably slightly better for them to be above, but IMO not worth a large effort.

**Comment by rossturk** [comment]
@ysndr although now I notice that the backquotes around 'script' and 'file' should be single quotes.

**Comment by ysndr** [comment]
fixed the quotes and some doc in #722

**Comment by ghudgins** [comment]
closing, LGTM
